app/controllers/users_controller.rb

```diff
@@ -46,6 +66,6 @@ def create_params
   end
 
   def update_params
-    params.require(:user).permit(:password)
+    params.require(:user).permit(:unconfirmed_email, :password)
```

Comment:
Please change param name to `:email`

Reply:
Breaks email edit, but leaving 'unconfirmed_email' leaks internal. Will get back to this later.

Reply:
Fixed!